### PR TITLE
Nut11 test vectors and nut fixes

### PR DESCRIPTION
**Pull request by thesimplekid**
Some test vectors for NUT11 but two things I think should be discussed and clarified in the nut before merging. 

1) Should the pubkey be the 33 byte key with a a prefix 02/03 or the 32 byte key without the prefix. I lean towards enforcing the 32 byte. 
2) What signature/signatures are required on the blinded messages when there are multiple proofs with different conditions. This one is a bit more complicated as there could also be multi-sig proofs. Simplest would be to require a single signature from any of the pubkeys in the proofs.

### CODE REVIEW

**Comment by callebtc** [comment]
> 1. Should the pubkey be the 33 byte key with a a prefix 02/03 or the 32 byte key without the prefix. I lean towards enforcing the 32 byte.

I think I agree. Two reasons come to mind:
- Secret derivation a la Bip32 also derives 32 bytes of randomness.
- Nostr keys are also 32 bytes, apparently ignoring the `02/03` parity prefix (I guess we're all assuming EC points on the lower half of the plane i.e. `02`)

File: 11.md [hunk 1]
```diff
@@ -16,40 +16,42 @@ If for a `Proof`, `Proof.secret` is a `Secret` of kind `P2PK`, the proof must be
...
+
 ```json
 [
-"P2PK", 
+  "P2PK",
   {
-    "nonce": "5d11913ee0f92fefdc82a6764fd2457a",
-    "data": "026562efcfadc8e86d44da6a8adf80633d974302e62c850774db1fb36ff4cc7198",
+    "nonce": "859d4935c4907062a6297cf4e663e2835d90d97ecdd510745d32f6816323a41f",
+    "data": "0249098aa8b9d2fbec49ff8598feb17b592b986e62319a4fa488a3dc36387157a7",
```
[the remark this answers is not in the document]
Comment (thesimplekid) [reply 11.md]: Yes, have updated the nut and test vectors

**Comment by callebtc** [comment]
Gorgeous work!

**Review by callebtc — APPROVED** [review]
utACK

will implement this later

**Pull request merged**